### ISSUE TITLE
time-consuming calculations 

### ISSUE THREAD

**Issue by pedrofeliz90** [dickreuter/Poker#154]
Hello, I'm testing the program to be able to sign, but in PS it is getting Sit Out, the calculations take time to do the action, what should I do to improve this?

**Comment by ryanwood1** [comment]
Hi.
Did you find an answer to improve the calculation speed?

**Comment by windowshopr** [comment]
Yeah there seems to be a lot of people having similar speed issues. Hopefully this gets resolved, seems like a cool project!
